keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -169,22 +195,20 @@ type SecurityContext struct {
 type SeccompProfile struct {
     // +unionDescriminator
     Type SeccompProfileType
-    // Use a predefined profile defined by the runtime.
-    // Most runtimes only support "default"
-    // +optional
-    RuntimeProfile *string
     // Load a profile defined in static file on the node.
     // The profile must be preconfigured on the node to work.
+    // LocalhostProfile must be 253 characters or less, and cannot be an 
+    // absolute nor a descending path.
```

Comment:
this length limit is still more restrictive than the current annotation validation... need to describe what happens when an annotation that cannot be copied into a field is encountered during a pod create request

Reply:
I removed the length limit. I could not find anywhere max length for annotations but managed to create pods with values north of 5000 chars, so I am assuming there is no hard limit.